server.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+require 'multi_json'
+require 'net/http'
+require 'uri'
+
+configure do
+  set :port, 5000
+end
+
+get '/' do
+  send_file 'index.html'
+end
+
+post '/user/verify' do
+  data = MultiJson.load(request.body.read)
+
+  url = URI.parse(data['apiUrl'])
+  req = Net::HTTP::Get.new(url.path)
+  req['Authorization'] = data['authHeader']
+
+  http = Net::HTTP.new(url.host, url.port)
+  http.use_ssl = true if url.scheme == 'https'
+  res  = http.request(req)
+
+  halt res.code.to_i, 'Error! Request failed.' unless res.code == '200'
+
+  MultiJson.dump(res.body)
```

Comment:
Could we just do `res.body` here without the `MultiJson.dump` part so it's consistent with what the Python implementation does? Then we always get the same representation in the front-end without having to do `JSON.parse` when dealing with Ruby but not for Python!